setup.py

```diff
@@ -34,7 +34,7 @@
 
 setuptools.setup(
     name='google-resumable-media',
-    version='0.2.3',
+    version='0.2.4',
```

Comment:
Let's hold off on a revision / release until another PR dedicated to that purpose (I can handle that).

Reply:
Done